releasetool/github.py

```diff
@@ -124,7 +124,13 @@ def get_installation_access_token(
 
 
 class GitHub:
-    def __init__(self, token: GitHubToken, use_proxy: bool = False) -> None:
+    def __init__(
+        self, maybe_token: Union[GitHubToken, str], use_proxy: bool = False
+    ) -> None:
+        if type(maybe_token) is str:
+            token = GitHubToken(cast(str, maybe_token), "Bearer")
+        else:
+            token = cast(GitHubToken, maybe_token)
```

Comment:
Is the `cast` needed if the token that was passed is already a `GitHubToken`?

Reply:
Let me see if type checks pass without 👍

Reply:
It does want the cast:

```
error: Incompatible types in assignment (expression has type "Union[GitHubToken, str]", variable has type "GitHubToken")
```